### PR TITLE
Added a link to the test setup document

### PR DESCRIPTION
**Pull request by RanRegev**

### CODE REVIEW

**Review by tomereli — APPROVED** [review]
I guess for now its OK - but the idea is to port the ppt to the README.md in Markdown. So please add a task for that and a TODO in the README.md before merging.

**Comment by RanRegev** [comment]
https://github.com/prplfoundation/prplMesh/issues/1263

**Review by VladyslavTupikin — APPROVED** [review]

**Comment by VladyslavTupikin** [comment]
@RanRegev Looks like you have forgot to sign commit.

File: tests/README.md [hunk 1]
```diff
@@ -50,6 +50,7 @@ The test environment prepares these before the tests are started.
 
 ## Description of tests
 
+TODO: translate this presentation into markdown format (.md)
```
Comment (arnout) [inline tests/README.md:53]: This looks pretty ugly when rendered. I also don't think we need it in the repository, the issue is sufficient.

**Review by arnout — APPROVED** [review]
Nitpick on the commit message: as explained in CONTRIBUTING.md, it should start with a component name, and it should be imperative, and the summary line should be brief. So: "tests/README.md: link to the test setup document".

**Pull request merged**